app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Show the user details by ID</h1>
\ No newline at end of file
+<div class="user_container">
+  <div class="user_img">
+    <%= image_tag("image1.jpg", width:'100vw') %>
```

Comment:
- [ ] It would be better if users can see 5 of their most recent posts. For example; `<% @user.recent_post.each do |post| %>`

Reply:
Hello bro @addod19  I'm pretty sure I have it already in the same file you have just mentioned on line 18. Please see below:
https://github.com/Whoistolu/blog-app/blob/d4e10db63e791a975a10c63ed6666dce556d32ff/app/views/users/show.html.erb#L18